framework/doc/content/source/meshgenerators/XYZMeshPlaneCutter.md

```diff
@@ -0,0 +1,68 @@
+# XYZMeshPlaneCutter
```

Comment:
what's the rationale for the XYZ prefix in the name?

the mesh line cutter only worked in the XY plane 

I m thinking PlaneCutterGenerator for now.
Maybe PlaneMeshCutterGenerator if we are planning to have PlaneBlock/Sideset/...CutterGenerators but I dont think so

Reply:
`PlaneMeshCutterGenerator` sounds okay.

Reply:
I m leaning towards CutMeshByPlaneGenerator now. 

I think that s a better naming convention. Though it s not widely followed